system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -326,34 +320,89 @@
 }
 
 .topsite-form {
+  $form-width: 300px;
+  $form-spacing: 32px;
+
+  .form-input-container {
+    max-width: $form-width + 3 * $form-spacing + $rich-icon-size;
+    margin: 0 auto;
+    padding: $form-spacing $form-spacing 40px;
+
+    .top-site-outer {
+      padding: 0;
+      margin: 24px 0 0;
+      margin-inline-start: $form-spacing;
+      pointer-events: none;
+      cursor: default;
+    }
+
+    .section-title {
+      text-transform: none;
+      font-size: 16px;
+      line-height: 19px;
+      color: $grey-60;
+      margin: 0 0 16px;
+    }
+  }
+
+  .fields-and-preview {
+    display: flex;
+  }
+
+  label {
+    font-size: $section-title-font-size;
+  }
+
   .form-wrapper {
-    margin: auto;
-    max-width: 350px;
-    padding: 15px 0;
+    width: 100%;
 
     .field {
       position: relative;
+
+      .clear-input-value-container {
+        position: absolute;
+        top: 16px;
+        width: 24px;
+        background: $white;
+        pointer-events: none;
+
+        &:dir(rtl) {
+          offset-inline-start: 1px;
+        }
+
+        &:dir(ltr) {
+          offset-inline-end: 1px;
+        }
+
+        .clear-input-value {
+          width: $icon-size;
+          height: $icon-size;
+          margin: 0 auto;
+          fill: $fill-secondary;
+          -moz-context-properties: fill;
+          background: url('#{$image-path}glyph-cancel-16.svg');
+          pointer-events: auto;
```

Comment:
`auto` doesn't seem right. Our other buttons have `pointer`

Reply:
The one in `about:preferences` doesn't have a pointer.